src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -105,6 +106,11 @@ define(function (require, exports, module) {
     function start(){
         window.addEventListener("message", _listener);
 
+        var autoUpdate = BrambleStartupState.ui("autoUpdate");
+        if(typeof autoUpdate === "boolean") {
```

Comment:
literally anything I believe. BrambleStartupState is just a simple map currently that doesn't do any type checking.